### PR TITLE
FIX: allow an admin to click on blank errors

### PR DESCRIPTION
**Pull request by LeoMcA**
Oops, in https://github.com/discourse/discourse/pull/5026 I forgot to show a string in the admin interface for blank errors so they can actually be clicked on and investigated:

screenshot from 2017-08-04 17-35-12

This fixes that!

### CODE REVIEW

File: app/serializers/incoming_email_details_serializer.rb [hunk 1]
```diff
@@ -16,7 +16,11 @@ def initialize(incoming_email, opts)
   EMAIL_RECEIVER_ERROR_PREFIX = "Email::Receiver::".freeze
 
   def error
-    @error_string
+    if @error_string.blank?
+      "Unrecognized Error"
```
Comment (ZogStriP) [inline app/serializers/incoming_email_details_serializer.rb]: Can this be I18nlized?

Comment (LeoMcA) [inline app/serializers/incoming_email_details_serializer.rb]: @ZogStriP done!

File: app/serializers/incoming_email_details_serializer.rb [hunk 2]
```diff
@@ -17,7 +17,7 @@ def initialize(incoming_email, opts)
 
   def error
     if @error_string.blank?
-      "Unrecognized Error"
+      I18n.t("emails.incoming.unrecognized_error")
```
Comment (ZogStriP) [inline app/serializers/incoming_email_details_serializer.rb]: You can shorten this to

```ruby
def error
  @error_string.presence || I18n.t("emails.incoming.unrecognized_error")
end
```

;)

Comment (LeoMcA) [inline app/serializers/incoming_email_details_serializer.rb]: Neat, I was wanting to do something in that form but couldn't work it out... now I know!
